README.md

```diff
@@ -9,11 +9,11 @@ Ember Sinon QUnit
 This addon integrates [Sinon](http://jhawk.co/sinonjs) & [Ember-QUnit](http://jhawk.co/ember-qunit) 
 via [Ember-Sinon](http://jhawk.co/ember-sinon), as inspired by [Sinon-QUnit](http://jhawk.co/sinon-qunit).
 
-Why not simply use Ember-Sinon? Two reasons:
+Why not simply use Ember-Sinon alone? Two reasons:
 
-1. Ember-Sinon does not handle cleanup of Ember-QUnit tests. Ember-Sinon-QUnit provides a test method 
-that wraps Ember-QUnit while making each test callback a sandboxed Sinon environment. All spies/stubs created
-via the sandbox will be automatically restored to their original methods at the end of each test.
+1. Ember-Sinon does not handle cleanup of Ember-QUnit tests. Ember-Sinon-QUnit automatically 
```

Comment:
I wonder if removing the `sandbox` terminology would help with overall clarity. It may be worth restructuring this so that I references the fact that [sinon automatically sandboxes itself now](https://sinonjs.org/guides/migrating-to-5.0), and be done with referring to sandboxes from then on. What do you think?

Reply:
Yea, that's fair. I'll do a once-over on the guides with this in mind.

Reply:
Done.